gen/types.go

```diff
@@ -55,6 +57,28 @@ func processTypeReplacements[C, I any](types drivers.Types, replacements []Repla
 	}
 }
 
+// matchString reports whether string a matches a pattern.
+// Pattern a can be either a literal string (case-insensitive comparison)
+// or a regular expression enclosed with /. Regex patterns are automatically made case-insensitive.
+func matchString(pattern, candidate string) bool {
```

Comment:
Everything is matched case-insensitively now. Is that alright?

Reply:
Also in practice, I don't think it is reasonable to have a "Users" and "users" table in the same database